### DISCUSSION TITLE
sauce service - test title names

### DISCUSSION THREAD

**Discussion by alexStrusberg** [webdriverio/webdriverio#7058] [Q&A]
Hi again :-) 

Regarding sauce labs , why does the test title on Sauce labs doesn't include the title of the describe + title of the it ?
This could be relay use full.

Thanks,
Alex S

**Comment by christian-bromann** [comment]
@alexStrusberg can you provide an example? How would a test look like and what should be the test name?

**Reply by alexStrusberg** [reply]
For example, in my case the "describe" is "Forgot password flow" and the "it" is "Should be able to reset password" or "Should not be able to reset password with invalid email"
The current titles are: Forgot password flow , Forgot password flow(1), Forgot password flow(2)

I would prefer: Forgot password flow -> Should be able to reset password
